### PR TITLE
C shuffle gallery

### PR DESCRIPTION
**Pull request by monalighosh**
Shuffle gallery component!

### CODE REVIEW

File: assets/scripts/evolution/shuffle_gallery.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+export default function() {
+
+	/******** Shuffle Gallery Component ********/
+
+	var links = document.getElementsByClassName("evo_c-shuffle-gallery__link");
```
Comment (jmatembu) [inline assets/scripts/evolution/shuffle_gallery.js]: Hey @monalighosh,

Please check for the existence of elements in the DOM before working with them. For example, at line 5, `links` is referring to some elements in the DOM, should it turn out that `links` is `undefined`, your code will break.

You could do:
```js
if (links) { 
//code
}
```
See this section on [writing safe JS code](https://github.com/BovAcademy-opensource/evolution-ui/blob/development/.github/CONTRIBUTING.md#writing-safe-javascript-code) for more details.

Comment (monalighosh) [reply assets/scripts/evolution/shuffle_gallery.js]: Hii...thanks for ur feedback...will make the necessary changes!

**Comment by monalighosh** [comment]
I have made the necessary changes to my JS file. Kindly have a look and let me know if there is anything. Thank you.

**Comment by GabrieleRomeo** [comment]
HI @monalighosh 

since you have also the PR "C shuffle gallery #65", I close this one.

Thanks

**Pull request closed without merging**